scripts/markdown_seo_lint.js

```diff
@@ -0,0 +1,219 @@
+const fs = require("fs");
+const yaml = require("js-yaml");
+const markdownlint = require("markdownlint");
+const path = require("path");
+
+const FRONT_MATTER_REGEX = /((^---\s*$[^]*?^---\s*$)|(^\+\+\+\s*$[^]*?^(\+\+\+|\.\.\.)\s*$))(\r\n|\r|\n|$)/m;
+
+function checkPageTitle(title) {
+    let error = null;
+    if (!title) {
+        error = "Missing page title";
+    }else if (title.trim().length === 0) {
+        error = "Page title is empty";
+    }else if (title.trim().length > 60) {
+        error = "Page title exceeds 60 characters";
+    }
+    return error;
+}
+
+function checkPageMetaDescription(meta) {
+    let error = null;
+    if (!meta) {
+        error = "Missing meta description";
+    }else if (meta.trim().length === 0) {
```

Comment:
Assigning `.trim().length` to a local variable, and just returning the string directly seems like it would clean up this function too.